src/goto-harness/function_call_harness_generator.cpp

```diff
@@ -185,6 +185,15 @@ void function_call_harness_generatort::validate_options()
       "--" FUNCTION_HARNESS_GENERATOR_FUNCTION_OPT};
 }
 
+std::size_t function_call_harness_generatort::require_one_size_value(
+  const std::string &option,
+  const std::list<std::string> &values)
+{
+  static auto const string_value = require_exactly_one_value(option, values);
+  // TODO replace this with string2optional
+  return narrow<std::size_t>(std::stoull(string_value));
```

Comment:
`string2optional` is available now (we rebased on develop already) so we should use it here

Reply:
:heavy_check_mark: